compiler/plugins/target/AMD-AIE/aie/test/unit_simple_flows_shim.mlir

```diff
@@ -1,30 +1,6 @@
 
 // RUN: iree-opt --split-input-file --amdaie-create-pathfinder-flows %s | FileCheck %s
 
-// CHECK-LABEL:   aie.device(xcvc1902) {
```

Comment:
Why remove this test?

Reply:
It uses PLIO: `aie.flow(%t23, North : 0, %t22, PLIO : 0)`